code/src/components/Actions.js

```diff
@@ -0,0 +1,60 @@
+import React from "react";
```

Comment:
Check this file for missing semi colons 😉

Reply:
fixed